internal/config/config.go

```diff
@@ -29,7 +26,7 @@ type Config struct {
 	Branch    string
 	CachePath string `yaml:"cache-path"`
 	OpenGL    string
-	Docker    bool
+	// Docker    bool
```

Comment:
Why is that config entry removed?

Reply:
I don't think the method of build should be part of the project config, it's a flag to the build command. The right way to persist this for projects is to use build-scripts / CICD.